mlflow/tracking/fluent.py

```diff
@@ -1776,25 +1777,15 @@ def setup_autologging(module):
     for module in list(set(LIBRARY_TO_AUTOLOG_FN.keys()) - {"pyspark", "pyspark.ml"}):
         register_post_import_hook(setup_autologging, module, overwrite=True)
 
-    # for pyspark, we activate autologging immediately, without waiting for a module import.
-    # this is because on Databricks a SparkSession already exists and the user can directly
-    #   interact with it, and this activity should be logged.
-    try:
+    if is_in_databricks_runtime():
+        # for pyspark, we activate autologging immediately, without waiting for a module import.
+        # this is because on Databricks a SparkSession already exists and the user can directly
+        #   interact with it, and this activity should be logged.
```

Comment:
What does `this activity` refer to?

Reply:
maybe it refers to autologging spark dataset information.

Reply:
Maybe?

Reply:
The comment refers to the activity of "pyspark" (not pyspark ml) autologging. (before I adding pyspark ML autologging the comment already existed.)
so it should refers to `_spark_autologging._listen_for_spark_activity`

Reply:
How about `register_post_import_hook`?

Reply:
Got it.